starter-core/src/main/java/io/micronaut/starter/feature/database/Oracle.java

```diff
@@ -22,17 +22,20 @@
 
 import io.micronaut.starter.feature.database.r2dbc.R2dbc;
 import io.micronaut.starter.feature.migration.MigrationFeature;
+import io.micronaut.starter.feature.testresources.TestResources;
 import jakarta.inject.Singleton;
 
 @Singleton
 public class Oracle extends DatabaseDriverFeature {
 
     public static final String NAME = "oracle";
```

Comment:
Should we use `NAME` in the other DB features?

We can then use Oracle.NAME in the tests